src/hist/intervals.py

```diff
@@ -124,26 +125,33 @@ def ratio_uncertainty(
         denom: Denominator or number of trials.
         uncertainty_type: Coverage interval type to use in the calculation of
          the uncertainties.
-         ``"poisson"`` (default) implements the Poisson interval for the
-         numerator scaled by the denominator.
-         ``"poisson-ratio"`` implements the Clopper-Pearson interval for Poisson
-         distributed ``num`` and ``denom`` where it is assumed that ``num`` and
-         ``denom`` are independent.
-         ``"efficiency"`` implements the Clopper-Pearson interval for Poisson
-         distributed ``num`` and ``denom``, with ``num`` assumed to be a
-         strict subset of ``denom``.
+         ``"poisson"`` (default) implements the Garwood confidence interval for
+         a Poisson-distributed numerator scaled by the denominator.
```

Comment:
No one is going to know what the Garwood confidence interval is, so if we use this terminology then there should be a direct link in the docs to a citable resource on it.

Reply:
I'll point to the docstring of `poisson_interval` which does have references